adafruit_ht16k33/ht16k33.py

```diff
@@ -81,16 +81,18 @@ def blink_rate(self, rate=None):
 
     @property
     def brightness(self):
-        """The brightness. Range 0-15."""
+        """The brightness. Range 0.0-1.0"""
         return self._brightness
 
     @brightness.setter
     def brightness(self, brightness):
-        if not 0 <= brightness <= 15:
-            raise ValueError('Brightness must be an integer in the range: 0-15')
-        brightness = brightness & 0x0F
+        if not 0.0 <= brightness <= 1.0:
+            raise ValueError('Brightness must be a decimal number in the range: 0.0-1.0')
+
         self._brightness = brightness
-        self._write_cmd(_HT16K33_CMD_BRIGHTNESS | brightness)
+        xbright = int(15 * brightness)
```

Comment:
Thanks for submitting this. May I suggest using `round()` instead of `int()`. The reason this works better in this situation is because if you pass a really close value like 0.99999 to brightness, `int()` will make it 14 and `round()` will make it 15.

Reply:
I made the change to switch from using int() to using round().